components/plurals/src/operands.rs

```diff
@@ -88,6 +89,10 @@ pub struct PluralOperands {
 }
 
 impl PluralOperands {
+    /// Construct a new [`PluralOperands`] given its various fields (documented on the struct)
+    pub fn new(i: u64, v: usize, w: usize, f: u64, t: u64, c: usize) -> Self {
```

Comment:
Nit: This type of constructor, with a bunch of homogeneous arguments, is extremely messy and hard to read. I'd prefer if we kept `Construct` on `Default::default()` without the constructor. Alternatively, we can make the struct exhaustive.